app/src/main/java/waiter/Router.java

```diff
@@ -1,23 +1,31 @@
 package waiter;
 
+import java.util.AbstractMap;
 import java.util.Map;
 
 public class Router {
 
     private final Map<String, Map<String, Map<String, String>>> routes;
     private final String noMethod = "No Method";
     private final String noUrl = "No Url";
+    Map.Entry<String, Map<String, String>> simpleGet = new AbstractMap.SimpleEntry<>("/simple_get", setResponseFields("200 OK", "Content-Length:", ""));
+    Map.Entry<String, Map<String, String>> noUrlFound = new AbstractMap.SimpleEntry<>(noUrl, setResponseFields("500 Internal Server Error", "Content-Length:", ""));
```

Comment:
What is the status code we're returning for `noUrlFound`? I don't think `500 Internal Server Error` is appropriate here. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404

Reply:
I'm curious what will happen if I try to access a URL that isn't registered? For example: `/blah`. Would I get a 500 internal server error?

Reply:
> What is the status code we're returning for `noUrlFound`? I don't think `500 Internal Server Error` is appropriate here. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404

I am using 500 for right now as a general place holder because responses such as 404 not found are future tickets/implementations, so I didn't want to do future work now if that makes sense

Reply:
> I'm curious what will happen if I try to access a URL that isn't registered? For example: `/blah`. Would I get a 500 internal server error?

It would indeed return a 500 internal server error

Reply:
I see. If this route is not needed, is there a reason we should keep it?

Reply:
As of right now, If I dont have it as this "default" then when the server gets a request it is not familiar with it would not be able to handle it in any form, resulting in the server hanging up

Reply:
I see. I'd suggest updating that to return 404 instead of 500. If 404 not found is slated for the future but we realized it's needed for the server to work then we should just implement it based on the HTTP specification. Right now we're in a weird spot where it's can sort of handle not found but not quite right. @matteeyao @elijahsullivan @jonmanahan thoughts?

Reply:
Totally agree w/ Steven. If the change does end up breaking your acceptance test, I would keep 500 to account for whatever route is being tested, but change default route to 404 to capture every other unidentified route.

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/17/commits/6fd747773a9adb7e5d4ef5e1a21cfb63ff761dcb)